content/docs/setup/kubernetes/helm-install.md

```diff
@@ -91,21 +91,175 @@ may be used in the same Helm operation.
 Helm charts expose configuration options which are currently in alpha.  The currently exposed options are explained in the
 following table:
 
-| Parameter | Description | Values | Default |
+<!-- AUTO-GENERATED-START -->
```

Comment:
@sdake do we need to add some comments for how does this table is generated with the python script at here?

Reply:
Yes we can add comments on line 93 - think it would make sense.